incubator/hnc/internal/validators/object_test.go

```diff
@@ -350,6 +350,20 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Deny creation of object with multiple non-negated treeSelect annotation",
```

Comment:
Do we have a test allowing multiple negated?

Reply:
Added